From 3929aafe753a8cd08bd6b76c67c8a9a0514e659c Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 17 May 2019 15:47:13 +0000
Subject: [PATCH 0882/1000] MANTIS 0054637 extension: Integration of new device
 H&S862 SR

codes.idl
V1.4.4
- add datastatus  DS_TYPES_CYPHER...

gmdevl.idl
V1.3
- add compType SCR = secure card reader

gmrdev.idl
V1.3
- add dataType DT_CYPHER


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1453 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/gmdev.idl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/gmdev.idl b/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
index 8784ca4c..c899e1ca 100644
--- a/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
@@ -6,7 +6,7 @@
  * Contains module gmdev, which is the platform-internal interface
  * definition for generic massai multi device components.
  *
- * @version 1.2
+ * @version 1.3
  *
  * @author Nicolas Bruns
  *
-- 
2.41.0.windows.1

